selfdrive/ui/qt/network/networking.cc

```diff
@@ -137,11 +137,22 @@ AdvancedNetworking::AdvancedNetworking(QWidget* parent, WifiManager* wifi): QWid
   main_layout->addWidget(back, 0, Qt::AlignLeft);
 
   ListWidget *list = new ListWidget(this);
+
+  // Auto tethering layout
+  const bool AutoTetheringEnabled = params.getBool("AutoTethering");
+  AutotetheringToggle = new ToggleControl(tr("Auto Tethering"), "", "", AutoTetheringEnabled);
```

Comment:
You can use `paramControl` instead which is a toggle + param manager, instead of you having to set the param yourself on connect